.travis.yml

```diff
@@ -16,6 +16,13 @@ before_install: |
     gpg --import --batch maven-artifact-signing-key.asc
   fi
 
+before_script:
+  - psql -U postgres -c 'CREATE DATABASE gtfs_lib_test;'
```

Comment:
Should we use the same programmatic create and drop that you used in osm-lib? It seems more portable (would work on a local build for example, as long as you had postgres running).

Reply:
I removed this